### PR TITLE
Improve homepage when there are no petitions

### PR DESCRIPTION
**Pull request by henryhadlow**
Couldn't sleep :/

### CODE REVIEW

File: app/assets/stylesheets/petitions/views/_home.scss [hunk 1]
```diff
@@ -87,7 +91,15 @@
   }
 }
 
-.local-to-you h2,
-.start-petition h2 {
-  @include bold-24;
+#local-to-you-heading,
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_home.scss]: We should not use the #anchors for headings. Better to use the classes as was previously done.

**Comment by alexander-bobin** [comment]
Generally LGTM. A shame we don't have any integration tests for it though.

File: app/assets/stylesheets/petitions/views/_home.scss (hunk 1 above)
Comment (henryhadlow) [inline app/assets/stylesheets/petitions/views/_home.scss]: Oops! Didn't mean to commit that. Will change that back.

**Comment by henryhadlow** [comment]
Just updated to address your previous line note.

Not that good at tests. We can revert this shortly after launch if you like.

**Comment by alexander-bobin** [comment]
OK. I'll merge on the understanding that the code looks good.

**Pull request merged**